test/api/container/JobContainerTest.php

```diff
@@ -48,4 +49,14 @@ public function testGcsOutputPath()
         $this->assertEquals('test_prefix/test/', $jobContainer->getOutputPath());
     }
 
+    public function testLiveStreamJobConfig()
+    {
+        $jobContainer = new JobContainer();
+        $jobContainer->job = new LiveStreamJobConfig();
+
+        $this->assertEquals($jobContainer->job instanceof LiveStreamJobConfig, true);
+        $jobContainer->job->streamKey = 'SomeKey';
+
+        $this->assertEquals($jobContainer->job->streamKey, 'SomeKey');
```

Comment:
This tests if setting LiveStreamJobConfig's streamKey attribute produces the expected result.